lampshade.go

```diff
@@ -437,3 +409,31 @@ func echo() []byte {
 	echo[tsSize] = frameTypeEcho
 	return echo
 }
+
+type pendingSession struct {
```

Comment:
Nit: Log entries like `Closing lampshade TCP connection: &lampshade.pendingSession{...` is a bit confusing. Maybe rename to `sessionConf` or so?

Reply:
OK